fvm_actor_utils/src/syscalls/fake_syscalls.rs

```diff
@@ -19,10 +19,13 @@ pub struct TestMessage {
 #[derive(Clone, Default, Debug)]
 pub struct FakeSyscalls {
     /// The root of the calling actor
-    pub root: Cid,
+    pub root: RefCell<Cid>,
     /// The f0 ID of the calling actor
```

Comment:
Could you update this comment to say "receiving actor"?

Reply:
Done, along with the ID line below it to avoid confusion with the caller ID